k8s/plugin/src/main.rs

```diff
@@ -20,12 +20,22 @@ struct CliArgs {
 
     #[clap(flatten)]
     args: resources::CliArgs,
+
+    /// Use namespace from the current `kubeconfig` context,
+    /// Takes precedence over --namespace or -n flag.
+    #[clap(global = true, long, short = 'c', default_value = "false")]
```

Comment:
We should hide this, since it's not meant to be used for now IIRC

```suggestion
    #[clap(global = true, hide = true, long = "namespace-from-context", default_value = "false")]
```

Reply:
Also should make it conflict with --namespace? Both supplied at the same time doesn't make sense

Reply:
I suppose user should not supply both at a same time. However, if they do then we give precedence to config file..We had to make it conflict as both will set ars.args.namespace otherwise

This flag anyway we can remove on the next major release. We wanted to remove namespace getting set to mayastor by default right?